### ISSUE TITLE
Root access to installed RancherOS without password

### ISSUE THREAD

**Issue by rfranke** [rancher/os#1453]
**RancherOS Version: (ros os version)**
0.5.0, 0.6.0, 0.7.1, 0.8.0-rc1

**Where are you running RancherOS? (docker-machine, AWS, GCE, baremetal, etc.)**
baremetal

The documentation for installation to disk says (https://docs.rancher.com/os/running-rancheros/server/install-to-disk/):
"After installing RancherOS to disk, you will no longer be automatically logged in as the rancher user. You’ll need to have added in SSH keys within your cloud-config file."

Since version 0.6.0 it is possible to log in through tty1 by just typing user name "rancher" and hit enter (no password), after installation to disk.

Shouldn't this be prevented and only login through ssh be allowed per default (cf. behavior up to 0.5.0)?

**Comment by rfranke** [comment]
The vulnerability appeared in 0.6.0-rc1. After install to and boot from disk the file /etc/shadow reads:
```
root:!:17106:0:99999:7:::
daemon:*:10933:0:99999:7:::
bin:*:10933:0:99999:7:::
sys:*:10933:0:99999:7:::
sync:*:10933:0:99999:7:::
mail:*:10933:0:99999:7:::
www-data:*:10933:0:99999:7:::
operator:*:10933:0:99999:7:::
nobody:*:10933:0:99999:7:::
sshd:*:::::::
rancher:FiZBnRzV.FSMQ::0:99999:7:::
docker:*:17106:0:99999:7:::
```

Until version 0.5.0 the line for rancher contained an asterisk instead of a password hash. Console login of rancher without password is disabled when changing the respective line in /etc/shadow to:
```
rancher:*::0:99999:7:::
```

@joshwget: did possibly something go wrong when you fixed #670?

**Comment by rfranke** [comment]
It works with the new 0.8.0-rc2. Thank you!
